app/graphql/resolvers/event_records_search.rb

```diff
@@ -51,6 +53,14 @@ def apply_take(scope, value)
     scope.take(value)
   end
 
+  def apply_data_provider(scope, value)
+    scope.joins(:data_provider).where(data_providers: { name: value })
+  end
+
+  def apply_data_provider_id(scope, value)
+    scope.joins(:data_provider).where(data_providers: { id: value })
+  end
```

Comment:
für event records könnte es wieder das problem geben mit "ist schon array und nicht mehr active record"?
wenn nach `"order": "listDate_ASC"` passiert das glaube ich. können mehrere filter kombiniert werden mit den neuen data providern? muss es vllt. auch rescues geben?

Reply:
werde ich prüfen

Reply:
`query { eventRecords(order: listDate_ASC, dataProviderId: 4){id,dataProvider{id, name}} }` =>

undefined method `joins' for #<Array:0x00007faac5b15928>
Did you mean?  join

Reply:
habe die Sortierung umgebaut, sodass eine ActiveRecordRelation zurückkommt, sodass die Scopeverkettung wieder funktioniert.

Reply:
ja perfekt. `ids` entnehmen und dann damit db seitig ordern. 👍